commitlint.config.ts

```diff
@@ -564,7 +564,7 @@ module.exports = {
                 'proper-revert-message': ({header, body}: {header: any, body: any}) => {
                     let offence = false;
 
-                    if (header.match(/^[Rr]evert/) !== null) {
+                    if (header.match(/^[Rr]evert ".+"/) !== null) {
```

Comment:
@realmarv let's add another test with this title (no body): `Revert "process overhaul" to fix CI`

Reply:
It's a failing test, right? because I don't see the reason of reverting

Reply:
Done